.github/pr-auto-comments.yml

```
@@ -0,0 +1,135 @@
+      body: >
+        When updating dependencies, take a moment to verify that there are not security
+        issues associated with any added or modified packages. If adding a new dependency,
+        ensure that it is totally necessary, and check the packages size is not too large,
+        as this will increase overall bundle size.
+
+    - id: environmental-vars
+      files: [ '.env', '*.env', '.env*' ]
+      body: >
+        You're making changes to the main server entry point. Please test the app thoroughly,
+        as any misconfigurations here may cause the app to stop functioning.
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)